code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1595,3 +1595,30 @@ var/list/rank_prefix = list(\
 /mob/living/carbon/human/proc/set_remoteview(var/atom/A)
 	remoteview_target = A
 	reset_view(A)
+
+/mob/living/carbon/human/proc/resuscitate()
+
+	if(!is_asystole() || has_organ(OP_HEART) || has_organ(BP_BRAIN))
```

Comment:
Why the check for if the person has a heart or brain?

Why would having these mean you can't resuscitate? Shouldn't it be that not having it should mean you can't resuscitate?

Reply:
It is also check if the organ is broken. Reagent stimulate heart muscles to work again, so you can be resuscitated.

Reply:
Also, if a person doesn't have brain, or have broken one, it can't be resuscitated.

Reply:
But, i forget to add another value. Thanks for reminding.
/mob/living/carbon/human/proc/has_organ(name, check_usablility = FALSE)
	var/obj/item/organ/external/O = organs_by_name[name]
	return (O && !O.is_stump() && (!check_usablility || O.is_usable()))

Reply:
I need to fix something here.

Reply:
Done